src/mesh/NodeDB.cpp

```diff
@@ -875,6 +875,12 @@ void NodeDB::updateFrom(const meshtastic_MeshPacket &mp)
 
         if (mp.rx_snr)
             info->snr = mp.rx_snr; // keep the most recent SNR we received for this node.
+
+        info->via_mqtt = mp.via_mqtt; // Store if we received this packet via MQTT
```

Comment:
This is good. I was wondering about how we could keep the information up to date with the latest packet delivery in the odd case of an mqtt node we know about that become available via lora range at some port, or vice-versa. Based on following this logic, updateFrom should keep us in sync with whatever state we have from the last packet, unless I'm mistaken.

Reply:
Indeed, it will get updated based on the value from each received packet.